inc/modules/export/epub/templates/epub3/opf.php

```diff
@@ -69,9 +71,11 @@
 		if ( ! empty( $meta['pb_author_file_as'] ) ) {
 			echo $meta['pb_author_file_as'];
 		} elseif ( ! empty( $meta['pb_author'] ) ) {
-			echo $meta['pb_author'];
+			$nameparser = new Parser();
+			$author = $nameparser->parse( $meta['pb_author'] );
```

Comment:
This library [can throw exceptions](https://github.com/davidgorges/HumanNameParser.php/blob/master/src/HumanNameParser/Parser.php#L188). What happens if it's a bad name? Try/Catch?